Lib/test/test_dummy_thread.py

```diff
@@ -253,3 +270,6 @@ def test_RaiseException(self, mock_print_exc):
         func = mock.Mock(side_effect=Exception)
         _thread.start_new_thread(func, tuple())
         self.assertTrue(mock_print_exc.called)
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
Just as a sidenote, while many test modules have these lines, they are not required to run, see https://devguide.python.org/runtests/

Reply:
Thanks! I'm not sure if this means I should get rid of these lines. Could you clarify that?

Reply:
No need to remove this - lot of the test modules have it. While it is not required to run the test(s) it is sometimes convenient to have it.